src/utils.h

```diff
@@ -0,0 +1,17 @@
+#include <sys/time.h>
+#include <time.h>
+
+#if defined(__APPLE__)
+#define HAVE_MACH_TIMER
+#include <mach/mach_time.h>
+#elif !defined(_WIN32) && !defined(_WIN64)
+#define HAVE_POSIX_TIMER
+#ifdef CLOCK_MONOTONIC
+#define CLOCKID CLOCK_MONOTONIC
+#else
+#define CLOCKID CLOCK_REALTIME
+#endif
+#endif
+
+/* Get system time*/
+static struct timeval rv_gettimeofday();
```

Comment:
Remove `static` qualifier since the function is invoked among compilation units.